languages/elm/exercises/concept/basics/.docs/introduction.md

```diff
@@ -0,0 +1,101 @@
+## Minimal compilable file
+
+### Modules
+
+Each file in Elm is a module, and must contain a `module` statement before all other code.
+
+Everything is scoped to the module, unless listed in `exposing`
+
+Module names must match their file name, so module `Greet` must be in file Greet.elm.
+
+```elm
+-- Define the Greet module, and expose the `greet` function
+module Greet exposing (greet)
+```
+
+```elm
+-- Define the Greet module, and expose everything
+module Greet exposing (..)
+```
+
+https://elm-lang.org/docs/syntax#modules
```

Comment:
The [introduction document specs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsintroductionmd) state:

> Links should be used sparingly, if at all. While a link explaining a complex topic like recursion might be useful, for most concepts the links will provide more information than needed so explaining things concisely inline should be the aim.

I think it is best to remove it from this document. This also goes for the other links in this document.